src/test/java/net/snowflake/ingest/SimpleIngestIT.java

```diff
@@ -315,4 +312,60 @@ private void verifyDefaultUserAgent(
       }
     }
   }
+
+  @Test
+  public void testConfigureClientHappyCase() throws Exception {
+    final String userAgentSuffix = "kafka-provider/NONE";
+    SimpleIngestManager manager = TestUtils.getManager(pipeName, userAgentSuffix);
+    ConfigureClientResponse configureClientResponse = manager.configureClient(null);
+    assertEquals(0L, configureClientResponse.getClientSequencer().longValue());
+  }
+
+  @Test
+  public void testConfigureClientNoPipeFound() throws Exception {
+    final String userAgentSuffix = "kafka-provider/NONE";
+    SimpleIngestManager manager = TestUtils.getManager("nopipe", userAgentSuffix);
+    try {
+      manager.configureClient(null);
+    } catch (IngestResponseException exception) {
+      assertEquals(
+          exception.getMessage(),
+          "HTTP Status: 404 ErrorBody: {\n"
```

Comment:
can we use object mapper to read this as a tree? It will give us JsonNode and it will be much easier to assert. 
```suggestion
JsonNode node = mapper.reader().readTree(exception.getMessage());
Assert node.get(code).getText()
.. and few more
```